packages/rabbitmq/test/docker.ts

```diff
@@ -168,7 +168,7 @@ export class RabbitMQContainer {
           { rabbitmqUrl },
           { logger: contextLogger(logger, 'enqueueTx') }
         );
-        await rabbitMqTxSubmitProvider.submitTx(txs[idx].txBodyUint8Array);
+        await rabbitMqTxSubmitProvider.submitTx({ signedTransaction: txs[idx].txBodyUint8Array });
```

Comment:
```suggestion
        await rabbitMqTxSubmitProvider.submitTx({ signedTransaction: txs[idx].txBodyHex });
```

Reply:
Had this change been tested?

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/393/commits/733679540e91c06b03b3a195cd1e5a352011a9df